src/builtin_echo.c

```diff
@@ -0,0 +1,31 @@
+#include "bash_ops.h"
+#include "env_singleton.h"
+#include "ft_printf.h"
+#include "libft.h"
+
+int	is_n_flag(t_command *cmd)
+{
+	if (ft_strcmp(cmd->argv->argv[1], "-n") == 0)
+		return 1;
+	return (0);
+}
+
+int	builtin_echo(t_command *cmd)
+{
+	int	count;
+
+	if (is_n_flag(cmd))
```

Comment:
why not use a proper bool? you call is_n_flag again, therefore calling ft_strcmp twice.